Index: app/src/main/java/com/petrus/asus/sportrnd/activity_forgetpass.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/petrus/asus/sportrnd/activity_forgetpass.java	(date 1554362913000)
+++ app/src/main/java/com/petrus/asus/sportrnd/activity_forgetpass.java	(date 1554363143307)
@@ -28,10 +28,10 @@
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_forgetpass);
 
-        inputEmail = (EditText) findViewById(R.id.email);
-        btnReset = (ImageButton) findViewById(R.id.btn_reset_password);
-        btnBack = (Button) findViewById(R.id.btn_back);
-        progressBar = (ProgressBar) findViewById(R.id.progressBar);
+        inputEmail = findViewById(R.id.email);
+        btnReset = findViewById(R.id.btn_reset_password);
+        btnBack = findViewById(R.id.btn_back);
+        progressBar = findViewById(R.id.progressBar);
 
         auth = FirebaseAuth.getInstance();
         btnBack.setOnClickListener(new View.OnClickListener() {
@@ -71,4 +71,4 @@
     }
 
 }
-/////////////////////////////////////////////////////////////////////////////////////////
\ No newline at end of file
+////////////////////////////////////////////////////////////////////////////////////////
\ No newline at end of file
